.github/workflows/end-to-end-tests.yml

```diff
@@ -0,0 +1,15 @@
+name: End-to-end tests
+on: [push]
+jobs:
+  cypress-run:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      # Install NPM dependencies, cache them correctly
+      # and run all Cypress tests
+      - name: Cypress run
+        uses: cypress-io/github-action@v2
+        with:
+          start: npm run test-servers
+          wait-on: npm run wait-for-test-server
```

Comment:
@joanise this is the workflow file for GitHub Actions CI. Let me know if you have any questions. Since workflow files are much more general purpose than, e.g., Travis-CI builds, there's a little bit more typing to get a basic test going. Luckily, Cypress provides its own action that makes setting up Cypress a bit easier.